api_tests/dry/multiple_peers.js

```diff
@@ -0,0 +1,138 @@
+const bob_comit_node_address = bob.config.comit.comit_listen;
+const charlie_comit_node_address = charlie.config.comit.comit_listen;
+
+let alice_swap_with_charlie_href;
+
+it("[Alice] Should be able to send a swap request to Bob", async () => {
+    await chai
+        .request(alice.comit_node_url())
+        .post("/swaps/rfc003")
+        .send({
+            alpha_ledger: {
+                name: alpha_ledger_name,
+                network: alpha_ledger_network,
+            },
+            beta_ledger: {
+                name: beta_ledger_name,
+                network: beta_ledger_network,
+            },
+            alpha_asset: {
+                name: alpha_asset_name,
+                quantity: alpha_asset_reasonable_quantity,
+            },
+            beta_asset: {
+                name: beta_asset_name,
+                quantity: beta_asset_quantity,
+            },
+            alpha_ledger_refund_identity: null,
+            beta_ledger_redeem_identity: alice_final_address,
+            alpha_expiry: alpha_expiry,
+            beta_expiry: beta_expiry,
+            peer: bob_comit_node_address,
+        })
+        .then(res => {
+            res.error.should.equal(false);
+            res.should.have.status(201);
+            swap_location = res.headers.location;
+            swap_location.should.be.a("string");
+            alice_reasonable_swap_href = swap_location;
+        });
+});
+
+it("[Alice] Should be able to send a swap request to Charlie", async () => {
+    await chai
+        .request(alice.comit_node_url())
+        .post("/swaps/rfc003")
+        .send({
+            alpha_ledger: {
+                name: alpha_ledger_name,
+                network: alpha_ledger_network,
+            },
+            beta_ledger: {
+                name: beta_ledger_name,
+                network: beta_ledger_network,
+            },
+            alpha_asset: {
+                name: alpha_asset_name,
+                quantity: alpha_asset_reasonable_quantity,
+            },
+            beta_asset: {
+                name: beta_asset_name,
+                quantity: beta_asset_quantity,
+            },
+            alpha_ledger_refund_identity: null,
+            beta_ledger_redeem_identity: alice_final_address,
+            alpha_expiry: alpha_expiry,
+            beta_expiry: beta_expiry,
+            peer: charlie_comit_node_address,
+        })
+        .then(res => {
+            res.error.should.equal(false);
+            res.should.have.status(201);
+            swap_location = res.headers.location;
+            swap_location.should.be.a("string");
+            alice_swap_with_charlie_href = swap_location;
+        });
+});
+
+it("[Alice] Should be in Start state after sending the swap request to Charlie", async function() {
+    await alice.poll_comit_node_until(
+        chai,
+        alice_swap_with_charlie_href,
+        "Start"
+    );
+});
+
+it("[Charlie] Shows the Swap as Start in /swaps", async () => {
```

Comment:
:question: Should we do the same for Alice's swap request to Bob?

Reply:
No. We test this already in the other file. I think we should use different quantities for this test and asset them though.